pom.xml

```diff
@@ -105,6 +104,8 @@
     <module>external/zeromq</module>
     <module>examples</module>
     <module>repl</module>
+    <!-- Invariant: Keep the assembly module at the bottom of the list -->
```

Comment:
Sure,  I will add the JIRA reference in the comment and will update the pull request